app/handler/spamd_result.py

```diff
@@ -101,6 +103,14 @@ def extract_from_headers(
                 spamd_result.set_spf_result(spf_result)
                 break
 
+        # parse the rspamd score
+        score_line = spam_entries[0]  # e.g. "default: False [2.30 / 13.00];"
+        spamd_result.rspamd_score = float(
+            score_line[(score_line.find("[") + 1) : score_line.find("]")]
+            .split("/")[0]
+            .strip()
+        )
```

Comment:
```suggestion
        try:
            score_line = spam_entries[0]  # e.g. "default: False [2.30 / 13.00];"
            spamd_result.rspamd_score = float(
                score_line[(score_line.find("[") + 1) : score_line.find("]")]
                .split("/")[0]
                .strip()
            )
        except (IndexError, ValueError):
            LOG.d("Could not parse rspamd score")
```

Reply:
I think it's better to raise exception so we get notified by sentry. (fail fast principle https://en.wikipedia.org/wiki/Fail-fast

Reply:
LOG.e then and we won't fail the delivery of the message if rspamd misbehaves.